Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/util/datetime/DurationUtil.java

```diff
@@ -0,0 +1,124 @@
+/*-
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2020 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+/**
+ * Various duration utility functions
+ *
+ * TO DO:
+ * Need to refactor further these duplicates
+ *
+ * ApromoreCore/Apromore-Extras/APMLogModule/src/main/java/org/apromore/apmlog/util/TimeUtil.java
+ * ApromoreEE/Dashboard/src/main/java/dashboard/util/Util.java
+ * ApromoreEE/FilterEE/src/main/java/org/apromore/plugin/portal/logfilteree/util/Util.java
+ */
+package org.apromore.util.datetime;
+
+import java.text.DecimalFormat;
+import java.time.temporal.ChronoUnit;
+
+import org.apromore.util.datetime.Constants;
+
+/**
+ * Various duration utility functions
+ */
+public final class DurationUtil {
+
```

Comment:
Same as above, please move the comment above the package to above the class definition.